### PR TITLE
added libinjection dependency

### PR DESCRIPTION
**Pull request by X4mp**
added libinjection dependency to requirements.txt

### CODE REVIEW

**Comment by afeena** [comment]
Thanks for the pr!

But we can't just add pylibinjection in our requirements, because if we go through setup, pip3 will give us the exception, because pylibinjection needs pip2:

```
diff --git a/requirements.txt b/requirements.txt
index 902ce73..a7c63c8 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -4,3 +4,4 @@ redis
 asyncio_redis
 uvloop
 pymongo
+pylibinjection
```
```
root@ubuntu:/opt/tanner# pip3 install -r requirements.txt
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 635, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 943, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 834, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (pip 8.1.1 (/usr/lib/python3/dist-packages), Requirement.parse('pip==7.1.0'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/pip3", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2927, in <module>
    @_call_aside
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2913, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2940, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 637, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 650, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 829, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pip==7.1.0' distribution was not found and is required by the application
```
Moreover, if we manually install pylibinjection via pip2, sqli_check.py will not work.

```
root@ubuntu:/opt/tanner# pip2 install pylibinjection
Collecting pylibinjection
  Downloading pylibinjection-0.2.4.tar.gz (68kB)
    100% |████████████████████████████████| 71kB 434kB/s
Collecting cython (from pylibinjection)
  Downloading Cython-0.25.2-cp27-cp27mu-manylinux1_x86_64.whl (6.4MB)
    100% |████████████████████████████████| 6.4MB 197kB/s
Building wheels for collected packages: pylibinjection
  Running setup.py bdist_wheel for pylibinjection ... done
  Stored in directory: /root/.cache/pip/wheels/fc/90/8c/05ef47b6ba316de9ac0793582f7d1bb00dccf70d4db7830072
Successfully built pylibinjection
Installing collected packages: cython, pylibinjection
Successfully installed cython-0.25.2 pylibinjection-0.2.4
You are using pip version 8.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@ubuntu:/opt/tanner# cd tanner/utils/
root@ubuntu:/opt/tanner/tanner/utils# python2 sqli_check.py UNION SELECT
Traceback (most recent call last):
  File "sqli_check.py", line 1, in <module>
    from libinjection import *
ImportError: No module named libinjection
```

**Comment by afeena** [comment]
This pr is related to #99 problem

**Comment by X4mp** [comment]
yes sure. I'll close this PR an open a new one with the fix

**Comment by X4mp** [comment]
reopening PR

File: tanner/emulators/sqli.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 import sqlite3
 import urllib.parse
+import pylibinjection
 from asyncio.subprocess import PIPE
```
Comment (afeena) [inline tanner/emulators/sqli.py]: subproccess is no longer necessary

**Comment by glaslos** [comment]
@afeena feel free to merge the PR if you are happy with the quality.

File: tanner/emulators/sqli.py [hunk 2]
```diff
@@ -28,21 +29,7 @@ def setup_db(self):
     @staticmethod
     @asyncio.coroutine
```
Comment (afeena) [inline tanner/emulators/sqli.py]: It is not async function anymore. So, we should delete decorator and also change function call

File: tanner/emulators/sqli.py [hunk 3]
```diff
@@ -28,21 +29,7 @@ def setup_db(self):
...
-        script_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../utils/sqli_check.py')
-        command = ['/usr/bin/python2', script_path, path]
-        res = yield from _run_cmd(command)
-        if res is not None:
-            try:
-                res = int(res.decode('utf-8'))
-            except ValueError:
-                res = 0
-        return res
+        return int(pylibinjection.detect_sqli(bytes(path, 'utf-8'))['sqli'])
```
Comment (afeena) [inline tanner/emulators/sqli.py]: Maybe it will be better, if we split this long line?

**Comment by X4mp** [comment]
I can do that later.
I can also split the line.

**Comment by afeena** [comment]
@X4mp Thanks :)

**Comment by afeena** [comment]
@X4mp  Great job, thanks :)

**Pull request merged**